src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseEndpoint.java

```diff
@@ -460,6 +460,35 @@ private void validateConstraints(final CollectionExerciseDTO collexDto) throws C
 
     }
 
+    /**
+     * PUT request for unlinking sample summary from a collection exercise
+     *
+     * @param collectionExerciseId the collection exercise to unlink from sample
+     * @param sampleSummaryId the collection exercise to unlink from collection exercise
+     * @return successful unlink message
+     * @throws CTPException            on resource not found
+     */
+    @RequestMapping(value = "/unlink/{collectionExerciseId}/sample/{sampleSummaryId}", method = RequestMethod.PUT)
+    public ResponseEntity<String> unlinkSampleSummary(
+            @PathVariable("collectionExerciseId") final UUID collectionExerciseId,
+            @PathVariable("sampleSummaryId") final UUID sampleSummaryId) throws CTPException {
+        log.debug("Entering unlinkSampleSummary with collectionExerciseID {} and sampleSummaryId {}",
+                collectionExerciseId, sampleSummaryId);
+
+        CollectionExercise collectionExercise = collectionExerciseService.findCollectionExercise(collectionExerciseId);
+        if (collectionExercise == null) {
+            throw new CTPException(CTPException.Fault.RESOURCE_NOT_FOUND,
+                    String.format("%s %s", RETURN_COLLECTIONEXERCISENOTFOUND, collectionExerciseId));
+        }
+
+        collectionExerciseService.removeSampleSummaryLink(sampleSummaryId, collectionExerciseId);
+
+        String responseMessage = "Successfully unlinked collection exercise "
+                + collectionExerciseId + " and sample " + sampleSummaryId;
+        return ResponseEntity.ok(responseMessage);
```

Comment:
I reckon we should get out of the habit of returning unstructured text from services. We should just return 204